### DISCUSSION TITLE
Favorite Coding Language

### DISCUSSION THREAD

**Discussion by infoyouth** [infoyouth/assignments#17] [Q&A]
What is your favorite **coding language** and why? Please share **one feature** that you find particularly useful."

**Comment by shweta-borganve** [comment]
My favorite coding language is Python. 
Because I started first coding language is python. and it is having Very Simple and Readable Syntax.
It is having Wide range of applications. Ex: Data analysis, web development, Machine learning, and Artificial Intelligence.

**Reply by infoyouth** [reply]
Very nice  @shweta-borganve

**Comment by DarshanBoraganve** [comment]
My favourite coding language is python because it is user friendly and it has lot of applications and built in modules ,machine learning is the feature that is useful in future .i like java also

**Reply by infoyouth** [reply]
Super @DarshanBoraganve

**Comment by Sanmati1234** [comment]
My favourite coding language is C and my first coding language .C is a powerful coding language that forms the foundation for many other language .it useful in embedded system,game development , system programming and learning other languages.and I like python also

**Reply by infoyouth** [reply]
Good answer @Sanmati1234

**Comment by shrutika-G** [comment]
My favourite coding language is java because  of the availability of enterprise-class libraries . it has lot of applications like web application ,data science,  web development and mobile app development

**Reply by infoyouth** [reply]
Very commandable @shrutika-G
